addons/website_forum/views/website_forum.xml

```diff
@@ -362,56 +371,70 @@
-                <p>If this approach is not for you, please respect the community and use Google+ communities instead.</p>
-                    <table class="table table-striped mt-4 bg-white">
-                        <tbody>
-                            <tr>
-                                <td class="faq-rep-item"><strong t-field="forum.karma_upvote"/></td>
-                                <td>upvote, add comments</td>
-                            </tr><tr>
-                                <td class="faq-rep-item"><strong t-field="forum.karma_downvote"/></td>
-                                <td>downvote</td>
-                            </tr><tr>
-                                <td class="faq-rep-item"><strong t-field="forum.karma_editor"/></td>
-                                <td>insert text link, upload files</td>
-                            </tr><tr>
-                                <td class="faq-rep-item"><strong t-field="forum.karma_user_bio"/></td>
-                                <td>your biography can be seen as tooltip</td>
-                            </tr><tr>
-                                <td class="faq-rep-item"><strong t-field="forum.karma_comment_unlink_own"/></td>
-                                <td>delete own comment</td>
-                            </tr><tr>
-                                <td class="faq-rep-item"><strong t-field="forum.karma_close_own"/></td>
-                                <td>flag offensive, close own questions</td>
-                            </tr><tr>
-                                <td class="faq-rep-item"><strong t-field="forum.karma_edit_all"/></td>
-                                <td>edit any post, view offensive flags</td>
-                            </tr><tr>
-                                <td class="faq-rep-item"><strong t-field="forum.karma_answer_accept_all"/></td>
-                                <td>accept any answer</td>
-                            </tr><tr>
-                                <td class="faq-rep-item"><strong t-field="forum.karma_comment_unlink_all"/></td>
-                                <td>delete any comment</td>
-                            </tr><tr>
-                                <td class="faq-rep-item"><strong t-field="forum.karma_close_all"/></td>
-                                <td>close any posts</td>
-                            </tr><tr>
-                                <td class="faq-rep-item"><strong t-field="forum.karma_unlink_all"/></td>
-                                <td>delete any question or answer</td>
-                            </tr>
-                        </tbody>
-                    </table>
+        <a t-attf-href="/forum/#{slug(forum)}/faq/karma">Why can other people edit my questions/answers?</a>
+    </t>
+</template>
+
+<!-- FAQ Karma Layout -->
+<template id="faq_karma" name="Karma">
+    <t t-call="website_forum.header">
+        <div t-field="forum.description" class="mb-4"/>
```

Comment:
Is not this strange to have this here?

Reply:
To have the karma layout in website_forum.xml or are you referencing something else ?

Reply:
I am referencing the line I am referencing, the forum description t-field.

Reply:
Ok, I just kept it because we're still in "About", but I can remove if needed.